container-scan/action.yaml

```diff
@@ -69,14 +59,123 @@ inputs:
   target:
     required: false
     description: Target build stage for multi-stage Docker builds
+  wiz-client-id:
```

Comment:
Because you are removing inputs and adding new required inputs, these changes should be breaking and create a major version

Reply:
Also, since this would be a breaking change, you could also take this opportunity to remove these vars

```
  MAX_MIND_USER:
    required: false
    description: MaxMind User ID
  MAX_MIND_LICENSE_KEY:
    required: false
    description: MaxMind License Key
  vfunction-file:
```

As we don't use vfunction anymore and the other env can be moved to `input.args` or the new input suggested below